src/main/java/com/gojek/beast/consumer/MessageConsumer.java

```diff
@@ -55,7 +55,7 @@ private Status pushToSink(ConsumerRecords<byte[], byte[]> messages, Instant poll
             final Instant deSerTime = Instant.now();
             ConsumerRecordConverter recordConverter = this.protoUpdateListener.getProtoParser();
             records = recordConverter.convert(messages);
-            statsClient.timeIt("kafkaConsumer.batch.deserialization.time,size=" + records.size(), deSerTime);
```

Comment:
In the polltime sent to Records constructor, the time taken to poll messages from kafka is also taken into account, but we are not pushing the time taken to poll data.

Reply:
Please add one time value for kafka poll time as well